registry_module.go

```diff
@@ -377,35 +376,19 @@ func (r *registryModules) CreateWithVCSConnection(ctx context.Context, options R
 	if err := options.valid(); err != nil {
```

Comment:
I think the org check should happen within here for GithubApp options

pseudocode?

```
if oauthToken
  return true
if installationID
  return validStringID(org)
return false
```

then when you choose your URL you would choose based on whether there is a oauthToken or not

Reply:
I moved the org check to the VCS Repo option valid function.